### PR TITLE
drupal 8.7

### PR DESCRIPTION
**Pull request by Natkeeran**
Do not merge!

## What does this Pull Request do?
* Installs drupal 8.7
* Enables Stand alone media url here `admin/config/media/media-settings`.  More info here: https://github.com/Islandora-CLAW/drupal-project/pull/27

## How should this be tested?
* Get the PR
* Vagrant up
* Verify that the setting is set
* Islandora 8 Sanity checks

@dannylamb

### CODE REVIEW

File: inventory/vagrant/group_vars/webserver/drupal.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ drupal_composer_dependencies:
   - "islandora/carapace:dev-8.x-1.x"
   - "islandora/openseadragon:dev-8.x-1.x"
   - "islandora/islandora_demo:dev-8.x-1.x"
-drupal_composer_project_package: "islandora/drupal-project:8.6.10"
+drupal_composer_project_package: "islandora/drupal-project:dev-drupal_87"
```
Comment (whikloj) [inline inventory/vagrant/group_vars/webserver/drupal.yml]: @Natkeeran I merged your PR and tagged it, so this can use `:8.7` now.

**Comment by dflitner** [comment]
I can't get this PR to complete vagrant up. It throws an error at trying to install islandora-demo. I think maybe it's due to this sourcing from an older version of the playbook so I'll try upgrading Drupal from the most recent.

**Comment by seth-shaw-unlv** [comment]
Are we able to close this due to #118?

**Comment by whikloj** [comment]
Yes, this is now replaced by #118

**Pull request closed without merging**